lib/rubocop/cop/style/guard_clause.rb

```diff
@@ -6,6 +6,10 @@ module Style
       # Use a guard clause instead of wrapping the code inside a conditional
       # expression
       #
+      # When condition with an `elsif` or `else` branch is allowed unless
```

Comment:
Probably you meant to say "A condition", as "when condition" seems wrong in this context.

Reply:
Indeed it is. Thank you very much.